test/functional/fixtures/es_archiver/date_nested/data.json

```diff
@@ -6,7 +6,7 @@
     "source": {
       "index-pattern": {
        "fields":"[]",
-        "timeFieldName": "@timestamp",
+        "timeFieldName": "nested.timestamp",
```

Comment:
:information_source: This is an interesting one. This data is only used in a test case, that tests if Discover properly show an error if building the request fails. This failed beforehand, since `@timestamp` was not a field at all in this index (pattern), thus the query building failed. With now using `dataView.isTimeBased()` that method already checks if the field exists and doesn't consider the index to be time based since it's not actually having a valid time field, thus no longer throwing an error and thus now failing the actual test.

I've made sure that the time field is now `nested.timestamp` (i.e. a timefield within a nested field), which I assume the test most likely anyway wanted to cover. cc @kertal